frontend/src/home/Home.tsx

```diff
@@ -313,23 +266,25 @@ class HomeComponent extends React.Component<Props, HomeState> {
     // remove existing toasts
     this.props.toastStack.forEach(t => this.props.removeToast(t.id));
 
-    let numVisibleWarnings: number = 0;
-    this.props.warnings.forEach(w => {
-      //ensuring we only propogate 5 toasts at a time
-      const yearIdx = this.props.schedule.years.indexOf(
-        convertTermIdToYear(w.termId)
-      );
-      if (!this.props.closedYears.has(yearIdx)) {
-        numVisibleWarnings++;
-        if (numVisibleWarnings <= 5) {
-          // add new toasts
-          this.props.addToast(w.message, {
-            appearance: "warning",
-            autoDismiss: true,
-          });
+    if (this.props.activePlan) {
```

Comment:
Only do this is we have an active plan. There is a time while the user plans are loading in the home screen that there is not an active plan, so we only want to render things that do not rely on an active plan while the user plans are loading

Reply:
nice